src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -320,36 +321,61 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 	 * @param props Map of properties that will be organized.
 	 * @returns A new map with different values appearing at the top and similar values appearing at the bottom.
 	 */
-	public sortPropertiesByDisplayValueEquivalency(props: Map<string, TablePropertiesMapEntry>, sortProperties: (props: Map<string, TablePropertiesMapEntry>) => Map<string, TablePropertiesMapEntry>): Map<string, TablePropertiesMapEntry> {
-		let unequalProperties: Map<string, TablePropertiesMapEntry> = new Map();
-		let equalProperties: Map<string, TablePropertiesMapEntry> = new Map();
+	public sortPropertiesByDisplayValueEquivalency(props: any[]): any[] {
```

Comment:
We should avoid `any` when we can - can you not use the `TablePropertiesMapEntry` type or something similar?